critical/back_end/vpcs.tf

```diff
@@ -23,9 +23,13 @@ locals {
   experience_cidr_block_public  = cidrsubnet(local.experience_cidr_block_vpc, 1, 0)
   experience_cidr_block_private = cidrsubnet(local.experience_cidr_block_vpc, 1, 1)
 
-  developer_cidr_block_vpc     = "172.42.0.0/16"
+  developer_cidr_block_vpc     = "172.56.0.0/16"
```

Comment:
This is the same as the digirati block - is that expected?

Reply:
Nope! Good catch - I actually fixed this before applying but forgot to commit that change - will push that update into master after this.